console/src/main/java/org/georchestra/console/ws/passwordrecovery/NewPasswordFormController.java

```diff
@@ -96,6 +96,7 @@ public String setupForm(@RequestParam(required = false) String token, Model mode
             formBean.setUid(uid);
 
             model.addAttribute(formBean);
+            model.addAttribute("pwdUtils", passwordUtils);
```

Comment:
what is ther purpose of adding this object to the model, if not used in the views afterwards ?

Reply:
ok got it, it's already being referred in the view, but since the object was not there, the display was broken.